omim2obo/main.py

```diff
@@ -246,7 +250,11 @@ def omim2obo(use_cache: bool = False):
             chr_id = '9606chr' + cyto_location
             # RO:0002525 (is subsequence of)
             # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0002525
-            graph.add((OMIM[mim_number], RO['0002525'], CHR[chr_id]))
+            b = BNode()
+            graph.add((b, RDF['type'], OWL['Restriction']))
+            graph.add((b, OWL['onProperty'], RO['0002525']))
+            graph.add((b, OWL['someValuesFrom'], CHR[chr_id]))
+            graph.add((OMIM[mim_number], RDFS['subClassOf'], b))
```

Comment:
This makes sure the relation is rendered correctly in OWL as an existential restriction.